lightning-rapid-gossip-sync/src/lib.rs

```diff
@@ -197,6 +197,7 @@ mod tests {
 				println!("{}", error_string);
 				return;
 			}
+			#[cfg(require_route_graph_test)]
```

Comment:
Is this necessary? If `require_route_graph_test` is not active, this line would already not be getting hit to begin with.

Reply:
Lol nvm, gotcha.